bindings/python/pymongoarrow/api.py

```diff
@@ -336,6 +337,10 @@ def write(collection, tabular):
             collection.insert_many(cur_batch)
         except BulkWriteError as bwe:
             raise ArrowWriteError(_transform_bwe(dict(bwe.details), cur_offset)) from bwe
+        except pymongo.errors.PyMongoError as pme:
+            raise ArrowWriteError(
+                {"writeErrors": [{"errmsg": pme._message, "index": cur_offset}]}
```

Comment:
`_message` is private, use `str(pme)` instead.

Reply:
Done.